src/mina-signer/mina-signer.ts

```diff
@@ -99,6 +99,30 @@ class Client {
     return PublicKey.toBase58(publicKey);
   }
 
+  /**
+   * Derives the public key corresponding to a given private key, ensuring compatibility with keys from the older [client_sdk](https://www.npmjs.com/package/@o1labs/client-sdk) libraries by converting the private key into the domain of the Pallas curve if necessary.
```

Comment:
hey @MartinMinkov I think it was other tools, not our own client sdk, that created out of domain keys, so would reword this comment to definitely not imply that

Reply:
the old client sdk just accepted them

Reply:
good point, fixed!